xml/schema.xml

```diff
@@ -22,7 +22,8 @@ betterPassword plugin schema in XML
         <field name="failed_login_time" type="T" >
             <NOTNULL/>
         </field>
+        <field name="passwords" type="X" />
```

Comment:
Yeah, I was going to use the settings, but as the plugin already had its own table, I thought it would be better to drop my data there as well, to decrease the pollution/orphan data.

ps: I'm curious if there's something to uninstall the plugins.

Reply:
I've replaced the fields by the user setting.